src/main/webapp/WEB-INF/springXML/services.xml

```diff
@@ -6,21 +6,21 @@
 
 
 
-    <bean id="AuthenticateService" class="com.books.services.AuthenticateService">
+    <bean id="AuthenticateService" class="com.books.services.AuthenticateServiceImpl">
         <constructor-arg ref="UserSQLDAO"></constructor-arg>
     </bean>
 
-    <bean id="encryptService" class="com.books.services.EncryptService"></bean>
+    <bean id="encryptService" class="com.books.services.EncryptServiceImpl"></bean>
 
-    <bean id="AuthorService" class="com.books.services.AuthorService">
+    <bean id="AuthorService" class="com.books.services.AuthorServiceImpl">
         <constructor-arg ref="AuthorSQLDAO"></constructor-arg>
     </bean>
 
-    <bean id="publisherService" class="com.books.services.PublisherService">
+    <bean id="publisherService" class="com.books.services.PublisherServiceImpl">
```

Comment:
Please, bind bean with interface, but not with class. In your case there is no sense in interfaces which are implemented by your services. It's not proper dependency injection.